.travis.yml

```diff
@@ -11,4 +14,18 @@ install:
 - yarn install --frozon-lockfile
 - yarn run bootstrap
 script:
-- yarn test:all
```

Comment:
This change can be reverted by 

```
git revert de1e16799709cba005432d6fa853b8e53b5ae064
```

The combination of matrix and job(beta) seem to do unexpected behavior.

Reply:
@azu, thanks much for your review. How did I miss your point (unexpected `LINT` value)...what I meant to do is:

```diff
@@ -20,13 +20,12 @@ script:
 matrix:
   exclude:
   - node_js: 4
-    env: LINT=true
+    env: LINT=false
   - node_js: stable
-    env: LINT=true
+    env: LINT=false
 jobs:
   include:
     - stage: lint
       if: env(LINT) = true
       script: yarn lint
       node_js: stable
```

which will resulted in:

[![My Travis CI build log](https://user-images.githubusercontent.com/679719/34723597-91e74d08-f58e-11e7-9f2f-756e109a3672.png)](https://travis-ci.org/0x6b/textlint/builds/326805622)

Anyway, as you pointed out, we don't have to introduce additional complexity for build. Will revert change to `.travis.yml`, update the PR, run CI build again, then merge.

Reply:
Year. Thanks for confirm.

> Will revert change to .travis.yml, update the PR, run CI build again, then merge.

OK.